src/main/java/io/dropwizard/elasticsearch/config/EsConfiguration.java

```diff
@@ -1,61 +1,37 @@
 package io.dropwizard.elasticsearch.config;
 
-import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.google.common.net.HostAndPort;
-import io.dropwizard.validation.ValidationMethod;
-import org.hibernate.validator.constraints.NotEmpty;
 
-import javax.validation.constraints.NotNull;
+import org.apache.http.HttpHost;
+
+import java.util.ArrayList;
 import java.util.Collections;
 import java.util.List;
-import java.util.Map;
+
+import javax.validation.constraints.NotNull;
 
 /**
  * Configuration class for Elasticsearch related settings.
  */
 public class EsConfiguration {
     @JsonProperty
     @NotNull
-    private List<HostAndPort> servers = Collections.emptyList();
-
-    @JsonProperty
-    @NotEmpty
-    private String clusterName = "elasticsearch";
-
-    @JsonProperty
-    private boolean nodeClient = true;
-
-    @JsonProperty
-    @NotNull
-    private Map<String, String> settings = Collections.emptyMap();
```

Comment:
There are quite few settings which might be interesting for the Low Level REST client.
Could we add them here?

https://www.elastic.co/guide/en/elasticsearch/client/java-rest/7.1/java-rest-low-usage-initialization.html
https://www.elastic.co/guide/en/elasticsearch/client/java-rest/7.1/java-rest-low-config.html

Also adding the possibility to add the Sniffer would be nice:
https://www.elastic.co/guide/en/elasticsearch/client/java-rest/7.1/_usage.html

Reply:
added a bunch more..  Not sure if did it properly..  Tests will be added soon.

Reply:
I added more tests now the coverage is higher and 100% on health checks.  

Could you comment more about what do you think is necessary to be covered in configuration? 

I added most of them listed in the documentation (but feel a bit hard to write test unless using testcontainer). Also I sort of feel it's a bit too complicate, maybe it's better to cover some simple common configs and then let sophisticated users do what they want with the `ManagedClient(client,sniffer)` constructor with their own client and sniffer..